src/database/repositories/GiveRepository.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-underscore-dangle */
+module.exports = class GiveRepository {
+  constructor(userModal) {
+    this.userModal = userModal;
+  }
+
+  async _give(field, fromID, toID, value) {
+    await this.userModal.updateOne({ id: fromID }, { $inc: { [field]: -value } });
+    await this.userModal.updateOne({ id: toID }, { $inc: { [field]: value } });
+  }
+
+  giveStars(fromID, toID, value) {
+    return this._give('estrelinhas', fromID, toID, value);
+  }
+
+  giveDemons(fromID, toID, value) {
+    return this._give('caçadas', fromID, toID, value);
+  }
+
+  giveAngels(fromID, toID, value) {
+    return this._give('anjos', fromID, toID, value);
+  }
+
+  giveSemigods(fromID, toID, value) {
```

Comment:
`Semideuses` in english is `demigods`, so if we are going to make the code english-only, lets use all variables in english
```suggestion
  giveDemigods(fromID, toID, value) {
```